tests/stdlib/tparsesql.nim

```diff
@@ -0,0 +1,376 @@
+discard """
+  file: "tparsesql.nim"
+  output: '''select
+  foo
```

Comment:
That's a horrible test. :-) Instead of the echo you should use ``doAssert`` so that it's visibible which output belongs to which input.

Reply:
Is there a doc that explains how testament works?

Reply:
Would some thing like this work `doAssert $parseSQL("SELECT foo FROM table;") == "select foo from table;"`